models/Bill.js

```diff
@@ -1,16 +1,21 @@
-const mongoose = require("mongoose");
+const mongoose = require('mongoose');
 const Schema = mongoose.Schema;
 const Receipt = require("./Receipt");
+const types = require('./types');
 
 const billSchema = new Schema({
-  category: String,
+  categories: {
+    type: String,
+    required: [true, 'Select the relevant categories for your bill'],
+    enum: types.billingCategory
+  },
   due_date: Date,
-  due_price: Number, // Should be Decimal128 (if it supports float)
+  due_price: Decimal128, // Should be Decimal128 (if it supports float)
```

Comment:
Does it actually work? @vsutinah

Reply:
Data type is supported by Mongoose (according to documentation), but apparently I fked up and didn't declare it the right away. Updated the Decimal128 type declarations to ensure that it works in the latest commit.